services/ui-src/src/components/export/ExportedModalDrawerReportSection.tsx

```diff
@@ -18,14 +18,15 @@ export const ExportedModalDrawerReportSection = ({
   const { report } = useContext(ReportContext);
   const { emptyEntityMessage } = exportVerbiage;
   const entities = report?.fieldData?.[entityType];
-  const entityCount = entities?.length;
+  const entityCount = entities ? entities?.length : undefined;
```

Comment:
Note: Hmm... I feel like `entities?.length` should return either the length count or undefined by default if `entities` is not defined. Is this ternary really necessary? Or could it just be the original? or maybe `entities?.length || 0` or `entities?.length || undefined`? I'm in a meeting so not running this locally lol.